### PR TITLE
Improve linux host adapter documentation

### PR DESCRIPTION
**Pull request by ericfranz**
https://osc.github.io/ood-documentation-test/linux_host_adapter_updates/installation/resource-manager/linuxhost.html

- Ensure that overview of adapter is at the top including the dependencies front and center
- Specify as beta and warn that it is not yet in production use at OSC
- Organize configuration of adapter by introducing the two ways to use it, and steps to configure for each

### CODE REVIEW

File: source/installation/resource-manager/linuxhost.rst [hunk 1]
```diff
@@ -1,89 +1,69 @@
...
-Approach #2: libcgroup cgroups
-------------------------------
 
-The libcgroup cgroups rules and configurations are a per-group resource limit where the group is defined in the examples at ``/etc/cgconfig.d/limits.conf``. The following examples limit resources of all tmux processes launched for the LinuxHost Adapter so they all share 700 CPU shares and 64GB of RAM. This requires setting ``tmux_bin`` to a wrapper script that in this example will be ``/usr/local/bin/ondemand_tmux``.
+Launch any software on target host
+----------------------------------
 
-Example of ``/usr/local/bin/ondemand_tmux``:
+1. Install on the target host a Singularity base OS image matching the OS of the target host. In the cluster configuration example below we use ``/opt/ood/linuxhost_adapter/centos_7.6.sif`` as the target host is running CentOS7.
+2. Add the cluster configuration file for the target host
```
Comment (treydock) [inline source/installation/resource-manager/linuxhost.rst:43]: Might be good to link to the cluster configuration section here. It took me a minute to realize it was further down the page when I was reading this.

File: source/installation/resource-manager/linuxhost.rst [hunk 2]
```diff
@@ -1,89 +1,69 @@
...
+This base image is used to provide an unprivileged PID namespace when launching the app. To make the rest of the software installed on the host available to launch within the image, most/all of the host file system is bind-mounted into the running container. For this reason many existing interactive apps will just work when launched by the LinuxHost adapter.
 
-   #!/bin/bash
-   exec tmux "$@"
+Launch specific application containers
+--------------------------------------
 
-Setup the cgroup limits at ``/etc/cgconfig.d/limits.conf``:
+1. Install the specific application containers on the target host.
+2. Add the cluster configuration file for the target host. Specify a default application container for ``singularity_image:`` and ``singularity_bindpath:`` even if you will override it.
```
Comment (treydock) [inline source/installation/resource-manager/linuxhost.rst:51]: Also might be good to link to cluster configuration section

File: source/installation/resource-manager/linuxhost.rst [hunk 3]
```diff
@@ -138,56 +118,108 @@ strict_host_checking
...
 
-The adapter pieces together several common Linux/HPC technologies to achieve behavior similar to what a scheduler offers.
+.. code-block:: none
 
-- ``ssh`` connects from the web node to a configured host such as a login node.
-- Specially named ``tmux`` sessions offer the ability to rediscover running jobs
-- ``singularity`` containerization provides a PID namespace without requiring elevated privileges that ensures that all child processes are cleaned up when the job either times out or is killed
-- ``timeout`` is used to set a 'walltime' after which the job is killed
-- ``pstree`` is used to detect the job's parent ``sinit`` process so that it can be killed
+   session     required      pam_exec.so type=open_session /etc/security/limits.sh
```
Comment (treydock) [inline source/installation/resource-manager/linuxhost.rst:156]: Should be mentioned this needs to go into a file used by the `sshd` PAM configs which on CentOS/RHEL default to `/etc/pam.d/password-auth-ac`.  Also needs to be included in the block showing position in chain as PAM order is very specific:

```
session     optional      pam_keyinit.so revoke
session     required      pam_limits.so
session     required      pam_systemd.so
session     required      pam_exec.so type=open_session /etc/security/limits.sh
session     [success=1 default=ignore] pam_succeed_if.so service in crond quiet use_uid
session     required      pam_unix.so
session     optional      pam_sss.so
```

Needs to come after `pam_systemd.so` that is also set to `required`.

**Comment by ericfranz** [comment]
@treydock addressed your comments. If you have no objections I will merge.

**Comment by treydock** [comment]
@ericfranz Looks good to me, go ahead and merge.

**Pull request merged**